src/test/java/com/google/firebase/auth/FirebaseUserManagerTest.java

```diff
@@ -27,270 +28,265 @@
+    UserRecord user = auth.createUserAsync(new CreateRequest()).get();
+    checkUserRecord(user);
     checkRequestHeaders(interceptor);
   }
 
   @Test
   public void testUpdateUser() throws Exception {
-    MockLowLevelHttpResponse response = new MockLowLevelHttpResponse();
-    response.setContent(TestUtils.loadResource("createUser.json"));
-    MockHttpTransport transport = new MockHttpTransport.Builder()
-        .setLowLevelHttpResponse(response)
-        .build();
-    FirebaseUserManager userManager = new FirebaseUserManager(gson, transport, credentials);
+    initializeAppForUserManagement(
+        TestUtils.loadResource("createUser.json"),
+        TestUtils.loadResource("getUser.json"));
+    FirebaseAuth auth = FirebaseAuth.getInstance();
+    FirebaseUserManager userManager = auth.getUserManager();
     TestResponseInterceptor interceptor = new TestResponseInterceptor();
     userManager.setInterceptor(interceptor);
-    // should not throw
-    userManager.updateUser(new UpdateRequest("testuser"), Utils.getDefaultJsonFactory());
+    UserRecord user = auth.updateUserAsync(new UpdateRequest("testuser")).get();
+    checkUserRecord(user);
     checkRequestHeaders(interceptor);
   }
 
   @Test
   public void testSetCustomAttributes() throws Exception {
-    MockLowLevelHttpResponse response = new MockLowLevelHttpResponse();
-    response.setContent(TestUtils.loadResource("createUser.json"));
-    MockHttpTransport transport = new MockHttpTransport.Builder()
-        .setLowLevelHttpResponse(response)
-        .build();
-    FirebaseUserManager userManager = new FirebaseUserManager(gson, transport, credentials);
+    initializeAppForUserManagement(TestUtils.loadResource("createUser.json"));
+    FirebaseAuth auth = FirebaseAuth.getInstance();
+    FirebaseUserManager userManager = auth.getUserManager();
     TestResponseInterceptor interceptor = new TestResponseInterceptor();
     userManager.setInterceptor(interceptor);
     // should not throw
     ImmutableMap<String, Object> claims = ImmutableMap.<String, Object>of(
         "admin", true, "package", "gold");
-    JsonFactory jsonFactory = Utils.getDefaultJsonFactory();
-    userManager.updateUser(new UpdateRequest("testuser")
-        .setCustomClaims(claims), jsonFactory);
+    auth.setCustomUserClaimsAsync("testuser", claims).get();
     checkRequestHeaders(interceptor);
 
     ByteArrayOutputStream out = new ByteArrayOutputStream();
     interceptor.getResponse().getRequest().getContent().writeTo(out);
+    JsonFactory jsonFactory = Utils.getDefaultJsonFactory();
     GenericJson parsed = jsonFactory.fromString(new String(out.toByteArray()), GenericJson.class);
     assertEquals("testuser", parsed.get("localId"));
     assertEquals(jsonFactory.toString(claims), parsed.get("customAttributes"));
   }
 
+  @Test
+  public void testRevokeRefreshTokens() throws Exception {
+    initializeAppForUserManagement(TestUtils.loadResource("createUser.json"));
```

Comment:
Could `initializeAppForUserManagement` be a setup() method that also encompasses the next 5 lines?

Reply:
The mock response is different in each test case (getUser.json, getUserError.json, listUsers.json etc.) Also half the tests in this file don't need it.

Reply:
FWIW, you could still pass the *.json file to the setup() method.

Reply:
Ok, I didn't add a setup method, but managed to significantly simplify this by moving rest of the user manager init code into the same helper method.